### DISCUSSION TITLE
What is the number for the latest Zettlr version to run on Windows 32?

### DISCUSSION THREAD

**Discussion by SN-CH** [Zettlr/Zettlr#4173] [Q&A] [answered]
[This page](https://github.com/Zettlr/Zettlr/issues/1552) says «Release 1.9.0 (February–April 2021): 32bit builds will be discontinued.». So, it should be possible to install and run 1.8.9 on Windows 32 (yes, I am a pour soul who ist forced to use it, there is nothing I can do to install Windows 64-bit on that computer), should it?

Well, it gives me «64-bit Windows is required» error message. I tried to install lower versions, down to 1.8.0 (downloaded from Zettlr's Github repo). All failed to install with exactly the same message.

**Comment by nathanlesage** [comment]
Are you sure you downloaded the 32bit version, or did you accidentally download the 64bit version? But in any case, there's no more support for that, so you're out of luck here.

**Comment by SN-CH** [comment]
@nathanlesage I realise you have stopped supporting 32bit a while ago.

But, I am looking for the highest old Zettlr version that does run on 32bit. For example, this [release page](https://github.com/Zettlr/Zettlr/releases/tag/v1.8.5) contains file  Zettlr-1.8.5.exe (which I expect to be 32-bit compatible, because the next release page for 1.8.6 contains only Zettlr-1.8.6-x64.exe, but my expectation is proved to be wrong).

1.8.5 is just an example. I went down to 1.8.0 and Windows 32 installer still refuses to launch it, because it is 64bit software.

**Reply by nathanlesage** [reply]
This is on me; I didn't include proper architecture labels in the installers until way after 32bit support was dropped. However, I just checked and until 2.0.0 32bit installers have been created.

It could be, however, that nobody with 32bit Windows actually used them and could therefore see if they indeed worked. This is another possibility: That 32bit Windows installers were simply always broken; I honestly don't know because I could never really test them.

**Reply by SN-CH** [reply]
I see. Thank you very much for this great piece of software any way. I have been using your program on other (Windows and Linux 64bit) computers for a while and I find it to be very useful.

**Comment by SN-CH** [comment, marked as answer]
For those who like me may be forced to run Zettlr on Windows 32bit. It was empirically revealed that the highest 32bit-compatible version is [1.7.5](https://github.com/Zettlr/Zettlr/releases/tag/v1.7.5).
